tests/test_lag_2.py

```diff
@@ -106,6 +132,72 @@ def verify_lag_lacp_timing(ptfhost, vm_name, lacp_timer, exp_iface):
     }
     ptf_runner(ptfhost, '/tmp', "lag_test.LacpTimingTest", '/root/ptftests', params=params)
 
+def verify_lag_minlink(
+    duthost,
+    vm_host,
+    lag_name,
+    peer_device,
+    intf,
+    neighbor_interface,
+    po_interfaces,
+    po_flap,
+    deselect_time,
+    wait_timeout = 30):
+
+    delay = 5
+    retries = wait_timeout / delay
+    try:
+        set_neighbor_interface(vm_host, neighbor_interface, shut=True)
+
+        # Let PortalChannel react to neighbor interface shutdown
+        time.sleep(deselect_time)
+
+        # Verify PortChannel interfaces are up correctly
+        for po_intf in po_interfaces.keys():
+            if po_intf != intf:
+                shell_retry(
+                    duthost,
+                    'bash -c "teamdctl %s state dump" | python -c "import sys, json; print json.load(sys.stdin)[\'ports\'][\'%s\'][\'runner\'][\'selected\']"' % (lag_name, po_intf),
+                    retries,
+                    delay)
+
+        # Refresh lag facts
+        lag_facts = duthost.lag_facts(host = duthost.hostname)['ansible_facts']['lag_facts']
+
+        # Verify lag member is marked deselected for the shutdown porta and all other lag member interfaces are marked selected
+        for po_intf in po_interfaces.keys():
+            logging.info("po: %s | %s | %s" % (po_intf, intf, lag_facts['lags'][lag_name]['po_stats']['ports'][po_intf]['runner']['selected']))
+            if po_intf != intf:
+                assert lag_facts['lags'][lag_name]['po_stats']['ports'][po_intf]['runner']['selected']
+            else:
+                assert not lag_facts['lags'][lag_name]['po_stats']['ports'][po_intf]['runner']['selected']
```

Comment:
I got assertion fail in my vstestbed...
No clue by now

Reply:
add debug log before assert?

Reply:
> add debug log before assert?

I plan to let the test sleep more time and check the interface's status in eos vm directly

Reply:
This is caused by a typo which cause the ansible command execution fail. I've fixed it